lib/customer.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'menu'
+require_relative 'text'
+require 'twilio-ruby'
+# require 'dotenv'
+# Dotenv.load
+
+
+class Customer
+  include Text
+
+  attr_reader :menu
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+    @order_total = 0
+  end
+
+  def select_dishes(dish, quantity=1)
+    fail "This dish is not on our menu" unless menu.menu_items.key?(dish)
+    quantity.times { |order| @order << dish }
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `order`. You can omit the argument if you don't care about it. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)